classes/PodsInit.php

```diff
@@ -822,6 +824,28 @@ public function register_assets() {
 
 		wp_set_script_translations( 'pods-dfv', 'pods' );
 
+		$config = [
+			'wp_locale'      => $GLOBALS['wp_locale'],
```

Comment:
:warning: Array double arrow not aligned correctly; expected 2 space(s) between "'wp_locale'" and double arrow, but found 6.

<sub>hash: 572bc9ae725aea8de4d36786a8ee4b2f</sub>